### DISCUSSION TITLE
Copyright Update

### DISCUSSION THREAD

**Discussion by VillainsRule** [socketio/socket.io#4585] [General] [answered]
It's 2023, and my client socket.io file still says 2014-2022 in the copyright.
This is a quick fix, I just wanted to note it.
<img width="241" alt="PoC" src="https://user-images.githubusercontent.com/101288516/211204073-5689fd4b-51fd-42b2-8899-7fdeb8f51d7b.png">

**Comment by darrachequesne** [comment, marked as answer]
Hi! Thanks for the heads-up, the date will be automatically updated in the next release.

Reference: https://github.com/socketio/socket.io-client/blob/18c6e0a6da23999ad128cfb47a90ff09148c84d6/support/rollup.config.umd.js#L7-L11
